src/orchestratord/src/controller/materialize.rs

```diff
@@ -92,6 +100,15 @@ pub struct Args {
     environmentd_internal_http_host_override: Option<String>,
     #[clap(long, default_value = "6879")]
     environmentd_internal_persist_pubsub_port: i32,
+    #[clap(long, default_value = "6881")]
+    environmentd_internal_balancer_port: i32,
```

Comment:
Shouldn't this have two ports? One for HTTP and one for SQL?

Reply:
oh, good point, i think i was just confused here, i'll fix